app/models/question.rb

```diff
@@ -227,23 +214,24 @@ def update_conditions(param_conditions)
   end
 
 
-  def saveCondition(value, option_id, remove_id = nil)
-    c = self.question_options.find(option_id).conditions.build
-    c.action_type = value['action_type']
-    c.remove_question_id = remove_id
+  def saveCondition(value)
+    c = self.conditions.build
```

Comment:
nice. I haven't seen `.build` on an association before. Will use it in the future